straxen/common.py

```diff
@@ -355,13 +355,23 @@ def pre_apply_function(data, run_id, target, function_name='pre_apply_function')
     return data
 
 
-def _load_function_file_from_home(function_file):
-    """For testing purposes allow this function file to be loaded from HOME"""
+def _overwrite_testing_function_file(function_file,
+                                     testing_folder='.straxen_testing'):
+    """For testing purposes allow this function file to be loaded from HOME/testing_folder"""
     home = os.environ.get('HOME')
-    if home is not None and os.path.exists(os.path.join(home, function_file)):
-        # For testing purposes allow loading from home
-        print(f'Using local function: {function_file}!')
-        function_file = os.path.join(home, function_file)
+    if home is None:
+        # Impossible to load from non-existent folder
+        return function_file
+
+    testing_file = os.path.join(home, testing_folder, function_file)
+
+    if os.path.exists(testing_file):
+        # For testing purposes allow loading from 'home/testing_folder'
+        warn(f'Using local function: {function_file} from {testing_file}! '
+             f'If you are not integrated testing on github you should '
+             f'absolutely remove this file. (See #559)')
```

Comment:
You can also check if you are running in a pytest session: 

https://docs.pytest.org/en/latest/example/simple.html#pytest-current-test-environment-variable

This might be more robust. Could also be useful at different places during our testing.

Reply:
Nice! Let me indeed change this! Should be done in 
5b6286b